LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -314,6 +314,13 @@
  */
 - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
 /**
+ @brief				Used for displaying the lock. The passcode view is added directly on the keyWindow.
+ @param superview   The superview where is to be presented, used to measure the center of the view.
+ @param hasLogout   Set to @c YES for a navBar with a Logout button, set to @c NO for no navBar.
+ @param logoutTitle The title of the Logout button.
```

Comment:
Writing some changes for this documentation made me realise a few things. 

Through this method, the lock isn't added to the `keyWindow` anymore, but to the `view` passed in, but the size of the lock is still fullscreen. For the default case of `showLockScreenWithAnimation` the logic remains the same, but  what is the true purpose of calling this method directly? Only to find a center? Or to also add the passcode inside the passed view? (To lock a specific part of the app, for example)

Depending on the answer, we might need to change the method name, or the documentation for it and the changelog.